Mathlib/Algebra/MvPolynomial/Basic.lean

```diff
@@ -1130,6 +1144,12 @@ theorem map_monomial (s : σ →₀ ℕ) (a : R) : map f (monomial s a) = monomi
 theorem map_C : ∀ a : R, map f (C a : MvPolynomial σ R) = C (f a) :=
   map_monomial _ _
 
+-- TODO: just make `map_ofNat` simp?
```

Comment:
I added these TODOs; I think we should attempt this before merging this PR

Reply:
#19388

Reply:
The bench results don't look so good